api/v1beta1/foundationdbbackup_types_test.go

```diff
@@ -21,8 +21,6 @@
 package v1beta1
 
 import (
-	"github.com/onsi/gomega"
```

Comment:
Can we separate these changes to a separate PR?

Reply:
Sure